rtc_tenclave/src/lib.rs

```diff
@@ -6,6 +6,7 @@
 #![feature(const_evaluatable_checked)]
 #![deny(clippy::mem_forget)]
 #![cfg_attr(not(test), no_std)]
+#![feature(impl_trait_in_bindings)]
```

Comment:
Since this is not necessary, and there's 2 open issues blocking stabilization, lets avoid this feature for now.

https://github.com/rust-lang/rust/issues/63065